assets/templates/account/signup.html

```diff
@@ -28,15 +28,15 @@ <h3>Personal information</h3>
                     {% include "snippets/bootstrap_form.html" with form=account_detail_form %}
 
                     <h3>Memberships</h3>
-                    {% include "snippets/bootstrap_form.html" with form=associationlink_form %}
+                    {% include "account/snippet_associations_form.html" with form=associationlink_form %}
 
                     <p>
                         <span class="text-danger">*</span> required
                     </p>
 
                     <h3>Terms of service</h3>
                     {# Todo: privacy documents #}
-                    <div class="offset-sm-3 col-sm-8">
+                    <div class="offset-sm-2 col-sm-10">
```

Comment:
Sorry :laughing: I changed the Bootstrap form template to `sm-2`/`sm-10` when I saw that Bootstrap itself uses that.